specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2023-09-01/capacityReservation.json

```diff
@@ -0,0 +1,1015 @@
+          "description": "List of the subscriptions that the capacity reservation group is shared with. **Note:** Minimum api-version: 2023-09-01. Please refer to https://aka.ms/computereservationsharing for more details."
+        }
+      }
+    },
+    "ResourceSharingProfile": {
+      "type": "object",
+      "properties": {
+        "subscriptionIds": {
+          "type": "array",
+          "description": "Specifies an array of Subscription resource Ids that capacity reservation group is shared with. The resource Id should be provided in form: /subscriptions/{subscriptionId}. **Note:** Minimum api-version: 2023-09-01. Please refer to https://aka.ms/computereservationsharing for more details.",
+          "items": {
+            "$ref": "../../../common-types/v1/common.json#/definitions/SubResource"
+          }
+        }
+      }
+    },
+    "CapacityReservationGroupProperties": {
+      "type": "object",
+      "properties": {
+        "capacityReservations": {
+          "type": "array",
+          "items": {
+            "$ref": "../../../common-types/v1/common.json#/definitions/SubResourceReadOnly"
+          },
+          "readOnly": true,
+          "description": "A list of all capacity reservation resource ids that belong to capacity reservation group."
+        },
+        "virtualMachinesAssociated": {
+          "type": "array",
+          "items": {
+            "$ref": "../../../common-types/v1/common.json#/definitions/SubResourceReadOnly"
+          },
+          "readOnly": true,
+          "description": "A list of references to all virtual machines associated to the capacity reservation group."
+        },
+        "instanceView": {
+          "$ref": "#/definitions/CapacityReservationGroupInstanceView",
+          "readOnly": true,
+          "description": "The capacity reservation group instance view which has the list of instance views for all the capacity reservations that belong to the capacity reservation group."
+        },
+        "sharingProfile": {
+          "$ref": "#/definitions/ResourceSharingProfile",
+          "description": "Specifies the settings to enable sharing across subscriptions for the capacity reservation group resource. Pls. keep in mind the capacity reservation group resource generally can be shared across subscriptions belonging to a single azure AAD tenant or cross AAD tenant if there is a trust relationship established between the AAD tenants. **Note:** Minimum api-version: 2023-09-01. Please refer to https://aka.ms/computereservationsharing for more details."
+        }
+      },
+      "description": "capacity reservation group Properties."
+    },
+    "CapacityReservationGroup": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/CapacityReservationGroupProperties"
+        },
+        "zones": {
+          "type": "array",
+          "items": {
+            "type": "string"
```

Comment:
>string [](http://example.com/codeflow?start=21&length=6)

are zones supposed to be strings or numbers? Let me consult the design guidelines...

Reply:
Ha, couldn't even find one!

Reply:
all zones in Compute are defined as string, so we will leave this one as is